packages/itwin/reports-config-widget/src/widget/ReportsConfigUiProvider.tsx

```diff
@@ -21,21 +21,16 @@ export const REPORTS_CONFIG_BASE_URL = "https://api.bentley.com";
  */
 export class ReportsConfigProvider implements UiItemsProvider {
   public readonly id = "ReportsConfigProvider";
-  private readonly _getAccessToken?: () => Promise<AccessToken>;
-  private readonly _baseUrl: string;
 
-  constructor(getAccessToken?: () => Promise<AccessToken>, baseUrl: string = REPORTS_CONFIG_BASE_URL) {
-    this._getAccessToken = getAccessToken;
-    this._baseUrl = baseUrl;
-  }
+  constructor(private _props: ReportsContainerProps) {}
```

Comment:
If everything in props is optional - make the props themselves optional.

Reply:
Done.